### PR TITLE
release: cut v0.62.0

### PR DESCRIPTION
**Pull request by simonpasquier**
## Description

Also includes a bump of Prometheus, Alertmanager & Thanos versions + regeneration of the "apply configuration" client code (because #5206 merged after #5243 while depending on an older main commit).

## Type of change

_What type of changes does your code introduce to the Prometheus operator? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [X] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Changelog entry

_Please put a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note

```

### CODE REVIEW

**Review by JoaoBraveCoding — APPROVED** [review]
lgtm 👍

File: pkg/operator/defaults.go [hunk 1]
```diff
@@ -18,14 +18,14 @@ import "github.com/prometheus/common/version"
...
-	DefaultAlertmanagerVersion = "v0.24.0"
+	DefaultAlertmanagerVersion = "v0.25.0"
 	// DefaultAlertmanagerBaseImage is a base container registry address for the prometheus alertmanager
 	DefaultAlertmanagerBaseImage = "quay.io/prometheus/alertmanager"
 	// DefaultAlertmanagerImage is a default image pulling address for the prometheus alertmanager
 	DefaultAlertmanagerImage = DefaultAlertmanagerBaseImage + ":" + DefaultAlertmanagerVersion
 
 	// DefaultThanosVersion is a default image tag for the Thanos long-term prometheus storage collector
-	DefaultThanosVersion = "v0.29.0"
+	DefaultThanosVersion = "v0.30.0"
```
Comment (PhilipGough) [inline pkg/operator/defaults.go:28]: v0.30.1 went out yesterday but I am not sure if you want to include the patch?

Comment (simonpasquier) [reply pkg/operator/defaults.go]: good call!

**Review by PhilipGough — APPROVED** [review]
lgtm

**Pull request merged**